cmd/skopeo/copy.go

```diff
@@ -41,17 +24,14 @@ func copyHandler(context *cli.Context) {
 	}
 	signBy := context.String("sign-by")
 
+	// TODO(runcom): unused digest??? Miloslav?
```

Comment:
Actually I don’t think there is a good reason for `copy` to write this to stdout at all. https://github.com/projectatomic/skopeo/pull/53 .